docs/manual/parallel_algorithms.qbk

```diff
@@ -271,6 +271,16 @@ __hpx__ provides implementations of the following parallel algorithms:
      [`<hpx/include/parallel_set_operations.hpp>`]]
 ]
 
+[table Heap operations (In Header: <hpx/include/parallel_algorithm.hpp>)
+    [[Name]     [Description]   [In Header]]
+    [[ [algoref is_heap] ]
+     [Returns `true` if the range is max heap.]
```

Comment:
Sure, just please add a link to cppreference.com like I did for the other algorithms, please (https://github.com/STEllAR-GROUP/hpx/commit/47e122eb73fb11c9cb1b42c0edab8f8cff972d2e).